testing/trino-testing/src/main/java/io/trino/testing/BaseConnectorTest.java

```diff
@@ -2919,6 +2920,7 @@ public void testInsert()
     public void testInsertForDefaultColumn()
     {
         skipTestUnless(hasBehavior(SUPPORTS_INSERT));
+        skipTestUnless(hasBehavior(SUPPORTS_CREATE_TABLE));
```

Comment:
this one isn't 100% accurate. Subclasses can provide their own `createTableWithDefaultColumns` which doesn't need to use Trino.